src/main/java/PageObjects/Railway/BookTicketPage.java

```diff
@@ -22,105 +28,64 @@ public class BookTicketPage {
     private final By infoTicketAmount = By.xpath("//table//td[count(//th[.='Amount']/preceding-sibling::th)+1]");
 
     // Elements
-    private WebElement getSelDepartDate() {
-
-        return Constant.WEBDRIVER.findElement(selDepartDate);
-    }
-
-    private WebElement getSelDepartFrom() {
-
-        return Constant.WEBDRIVER.findElement(selDepartFrom);
-    }
-
-    private WebElement getSelArriveAt() {
-
-        return Constant.WEBDRIVER.findElement(selArriveAt);
-    }
-
-    private WebElement getMsgBookSuccessfully() {
-
-        return Constant.WEBDRIVER.findElement(msgBookSuccessfully);
-    }
-
+    private WebElement getSelDepartDate() { return Constant.WEBDRIVER.findElement(selDepartDate);}
+    private WebElement getSelDepartFrom() { return Constant.WEBDRIVER.findElement(selDepartFrom);}
+    private WebElement getSelArriveAt() { return Constant.WEBDRIVER.findElement(selArriveAt);}
     private WebElement getSelSeatType() {
         return Constant.WEBDRIVER.findElement(selSeatType);
     }
-
     private WebElement getSelTicketAmount() {
         return Constant.WEBDRIVER.findElement(selTicketAmount);
     }
-
-    private WebElement getBtnBookTicket() {
-        return Constant.WEBDRIVER.findElement(btnBookTicket);
-    }
-
+    private WebElement getBtnBookTicket() { return Constant.WEBDRIVER.findElement(btnBookTicket);}
+    public WebElement getMsgBookSuccessfully() { return Constant.WEBDRIVER.findElement(msgBookSuccessfully);}
     private WebElement getInfoTicketDepart() {
         return Constant.WEBDRIVER.findElement(infoTicketDepart);
     }
-
     private WebElement getInfoTicketArrive() {
         return Constant.WEBDRIVER.findElement(infoTicketArrive);
     }
-
     private WebElement getInfoTicketSeatType() {
         return Constant.WEBDRIVER.findElement(infoTicketSeatType);
     }
-
     private WebElement getInfoTicketDepartDate() {
         return Constant.WEBDRIVER.findElement(infoTicketDepartDate);
     }
-
     private WebElement getInfoTicketAmount() {
         return Constant.WEBDRIVER.findElement(infoTicketAmount);
     }
 
     // Methods
-
-    public void bookTicket() {
-
+    public void bookTicket(int departDate, String departFrom, String arriveAt, String seatType, String ticketAmount) {
```

Comment:
This is `bookTicket` method so the function of it is 'book ticket', do not check anything inside of it